bash-completion/Makefile.am

```diff
@@ -0,0 +1,2 @@
+bashcompdir = @bashcompdir@
```

Comment:
This line should be unnecessary (automake automatically produces it).

Also, given the fact that this file will only contain the following line, I would move it to the top-level Makefile.am instead of using `include`.

Reply:
Fixed.